lms/djangoapps/philu_overrides/migrations/0001_initial.py

```diff
@@ -11,14 +11,10 @@ class Migration(migrations.Migration):
 
     def add_history_columns(apps, schema_editor):
         cursor = connection.cursor()
-        tables = ['enterprise_historicalenterprisecustomer', 'enterprise_historicaluserdatasharingconsentaudit']
-        for t in tables:
-            query = 'ALTER TABLE %s ADD start_date DATETIME NULL' % t
-            cursor.execute(query)
-            query = 'ALTER TABLE %s ADD end_date DATETIME NULL' % t
-            cursor.execute(query)
-            query = 'ALTER TABLE %s ADD history_change_reason VARCHAR(100) NULL' % t
-            cursor.execute(query)
+        query = 'ALTER TABLE enterprise_historicalenterprisecustomer ADD start_date DATETIME NULL'
```

Comment:
Check if we can execute single query for both columns

Reply:
Yes we can, here is the modified code
`ALTER TABLE enterprise_historicalenterprisecustomer ADD start_date DATETIME NULL, end_date DATETIME NULL;`